src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+.nav {
+  margin-right: 40px;
+  margin-left: 50px;
+  background-color: white;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav__picture {
+  justify-content: left;
+  flex: 1;
+}
+
+.nav__item {
+  justify-content: right;
+  padding: 0 10px;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 60px;
+  color: black;
+}
+
+.nav__link:visited {
+  color: blue;
```

Comment:
I wrote blue, but I meant the color #00acdc😅

Reply:
.nav__link:visited must be at the end of the link styles. Because it won’t work in your case